src/main/resources/com/oltpbenchmark/benchmarks/tpch/generation/dists.dss

```diff
@@ -0,0 +1,836 @@
+#
```

Comment:
@timveil Should this file be in `src/main/resources/benchmarks/tpch`?

Reply:
Shall this be re-located?

Reply:
Yes.

Reply:
resolved